v63005/inref/gtm5574.m

```diff
@@ -33,7 +33,8 @@
 neg
 	set %DH=n
 	set %DO=n
-	set %DL=1
+	kill %DL
+	set:($random(1)) %DL=1+$random(16)
```

Comment:
$random(1) -> $random(2)
$random(1) always returns 0 so the set command will never execute